manifests/platform.yaml

```diff
@@ -27,234 +27,234 @@ metadata:
 spec:
   sources:
     charts:
-    - name: csm-algol60
-      type: repo
-      location: https://artifactory.algol60.net/artifactory/csm-helm-charts/
+      - name: csm-algol60
```

Comment:
Does the extra tab in the whole file break the yaml formatting?

Reply:
Fixed.